crates/wasmtime/src/config.rs

```diff
@@ -606,6 +612,9 @@ impl Config {
     ///
     /// [proposal]: https://github.com/webassembly/bulk-memory-operations
     pub fn wasm_bulk_memory(&mut self, enable: bool) -> &mut Self {
+        if !enable {
+            assert!(!self.features.reference_types, "'reference_types' requires 'bulk_memory', cannot disable 'bulk_memory' without disable 'reference_types' first");
```

Comment:
Could we do the same sort of deferred validation for the combination of reference types and bulk memory that we do for compiler options? I'm somewhat uncomfortable with this assert existing, given that we have the right API signatures elsewhere to return the error.

Reply:
Fixed.